tests/console/force_cron_run.pm

```diff
@@ -30,13 +30,15 @@ sub settle_load {
 
 sub run {
     select_console 'root-console';
-
     # show dmesg output in console during cron run
     assert_script_run "dmesg -n 7";
 
     # Make sure there's no load before we trigger one via cron.
     settle_load;
     my $before = time;
+    # "bash -x /usr/lib/cron/run-crons" doesn't work really on a running system with online duration shorter than 15 minutes
+    # but what we can do is to disable all systemd timers. So keep bash -x /usr/lib/cron/run-crons for now
```

Comment:
why should we disable the systemd timers? I thought the ideas was to explicitly trigger the timers which do basically the same as the run-crons command would do (on older systems). Second question then is if we should still call run-cron on top.

Reply:
You can not trigger a timer (at least I am not aware of the possibility, even after studying the manpages), but only the service "behind" the trigger. I.e. if you have an hourly timer, it may still trigger again 5 minutes later.
This may be sufficient, e.g. it is likely running btrfs balance twice in a row is (mostly) a noop for the second run. There may be other triggers which cause the same amount of work each run.

Reply:
@Zaoliang @rwx788 I don't understand why do you want to disable services/timers?

@StefanBruens yes, I meant trigger services behind timers. And yes, I guess the idea is to trigger services explicitly and wait for their completion once. Any subsequent calls *should* be no-ops and the previous approach seems to have ensured this in principle.

Reply:
disabling might be the *only* way. As the trigger time is relative to the last time it ran, the monthly btrfs-balance.timer might hit in a saved HDD pretty much every test on a given day.

Reply:
But previously we also had the case that balancing could be triggered again but still a second run should not cause problems. Maybe we just need to improve how these problems show up in test results

Reply:
Then you are only talking about btrfs. So let's make this specific about this.

if rpm -q btrfsmaintenance ; then systemctl start btrfs-balance.service; fi
+ wait

Reply:
I guess this is a must we should do, as btrfs balancing is the one hitting us most, I haven't seen issues cause by other services.

Reply:
```
$ rpm -ql btrfsmaintenance
/usr/lib/systemd/system/btrfs-balance.service
/usr/lib/systemd/system/btrfs-balance.timer
/usr/lib/systemd/system/btrfs-defrag.service
/usr/lib/systemd/system/btrfs-defrag.timer
/usr/lib/systemd/system/btrfs-scrub.service
/usr/lib/systemd/system/btrfs-scrub.timer
/usr/lib/systemd/system/btrfs-trim.service
/usr/lib/systemd/system/btrfs-trim.timer
/usr/lib/systemd/system/btrfsmaintenance-refresh.path
/usr/lib/systemd/system/btrfsmaintenance-refresh.service
…
```